rules/falco_rules.yaml

```diff
@@ -1902,6 +1902,31 @@
   priority: INFO
   tags: [container, cis, mitre_privilege_escalation, mitre_lateral_movement]
 
+# These capabilities were used in the past to escape from containers
+- macro: excessively_capable_container
```

Comment:
I think `CAP_SYS_BPF` can also be abused to escape container.

Reply:
Added it